spydra/src/main/java/com/spotify/spydra/submitter/runner/SubmissionCliParser.java

```diff
@@ -88,6 +89,14 @@ public SpydraArgument parse(String[] args) throws IOException {
           sanitizeJobId(cmdLine.getOptionValue(CliConsts.JOBNAME_OPTION_NAME)));
     }
 
+    if (cmdLine.hasOption(CliConsts.LABELS_OPTION_NAME)) {
+      String labels = StringUtils.join(cmdLine.getOptionValues(CliConsts.LABELS_OPTION_NAME), ",");
+      spydraArgument.getSubmit().getOptions().compute(SpydraArgument.OPTION_LABELS,
+          (key, existingLabels) -> (existingLabels == null) ? labels : existingLabels + '.' + labels);
+      spydraArgument.getCluster().getOptions().compute(SpydraArgument.OPTION_LABELS,
+          (key, existingLabels) -> (existingLabels == null) ? labels : existingLabels + '.' + labels);
```

Comment:
Change of separator to `.` compared to few lines above. Could you clarify the intent?

Reply:
good catch =)